mep/people/templates/people/member_card_detail.html

```diff
@@ -80,11 +80,12 @@
             </tr>
         </thead>
         {% for event in events %}
-        <tr>
+        <tr id="{{ event.work.slug }}_{{event.partial_start_date }}">
```

Comment:
could there be collision here if there are two partial dates for that both evaluate to the same year (no month/day) for the same work?

Reply:
Possibly? Also possible to have different events with the same work (e.g. borrow and purchase, or purchase/borrow multiple volumes or issues of the same work).

Any thoughts on how to make more unique? How big of a problem is it when the id isn't unique?

Reply:
what is the id used for? it's invalid html if there are duplicate ids, but the page will obviously still render...not sure what actually ends up happening to the DOM

Reply:
@thatbudakguy id is used for an anchor link so we can highlight the relevant event when we link from the work card list to the card detail view.

That was what I thought, too - page will still render even if it's invalid. I think the duplicate ids get ignored (not sure if the first one or last one takes precedence, or if it depends on the browser). Unless we can come up with a simple way to make them unique, I'm inclined to leave it for now.

Reply:
why not add pk?

Reply:
@thatbudakguy I was going for readable, but that seems like an obvious solution. Should I _just_ use pk? It will be much shorter